src/ACEtk/block/details/BaseDistributionData/src/generateBlocks.hpp

```diff
@@ -0,0 +1,13 @@
+void generateBlocks() {
+
+  std::size_t nr = static_cast< std::size_t >( this->XSS( 1 ) );
+  std::size_t ne = static_cast< std::size_t >( this->XSS( 1 + 2 * nr + 1 ) );
+  auto begin = this->begin();
+  auto data = begin + 2 * nr + 1;
+  auto end = data + + 2 * ne + 1;
```

Comment:
Remove second + in `data +  + 2`.

Reply:
This is changed in a later PR. It now says:
```
auto end = this->iterator( 2 + 2 * nr + 2 * ne + 1 );
```